docs-chef-io/content/inspec/resources/mongodb_session.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "mongodb_session resource"
+draft = false
+gh_repo = "inspec"
+platform = "os"
+
+[menu]
+  [menu.inspec]
+    title = "mongodb_session"
+    identifier = "inspec/resources/os/mongodb_session.md mongodb_session resource"
+    parent = "inspec/resources/os"
++++
+
+Use the `mongodb_session` Chef InSpec audit resource to run MongoDB command against a MongoDB Database.
+
+## Availability
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+## Syntax
+
+A `mongodb_session` resource block declares the `user`, `password`, 'database' to use for the session, and then the command to be run:
+
+  describe mongodb_session(user: "username", password: "password", database: "test").query(key: value) do
+    its("params") { should match(/expected-result/) }
+  end
+
+where
+
+- `mongodb_session` declares a user, password and database, connecting locally, with permission to run the query.
+- `query` contains the query to be run.
+- `its("params") { should eq(/expected-result/) }` compares the results of the query against the expected result in the test
+
+### Optional Parameters
+
+`mongodb_session` InSpec resource accepts `user`, `password`, `host`, `port`, `auth_source`, `auth_mech`, `ssl`, `ssl_cert`, `ssl_ca_cert`, `auth_mech_properties`.
```

Comment:
Several of these options are not documented.

Reply:
Thanks, @clintoncwolfe. I have updated the docs.